sqlitecluster/SQLite.h

```diff
@@ -23,7 +23,7 @@ class SQLite {
     // maxRequiredJournalTableID: This is the maximum journal table ID that we'll verify. If it's -1, we'll only verify
     //                            'journal' and no numbered tables.
     SQLite(const string& filename, int cacheSize, int autoCheckpoint, int maxJournalSize, int journalTable,
-           int maxRequiredJournalTableID);
+           int maxRequiredJournalTableID, string synchronous = "false");
```

Comment:
Rather than having a default one set here, I'd suggest just allowing this to remain unset -- and we only call the `PRAGMA` statement if it's not set.  So, we just use SQLite's default if unspecified on the command line, and only override that default if specified.